include/rocksdb/perf_level.h

```diff
@@ -13,8 +13,8 @@ namespace rocksdb {
 
 // How much perf stats to collect. Affects perf_context and iostats_context.
 
-enum PerfLevel : char {
-  kUninitialized = -1,            // unknown setting
+enum PerfLevel : unsigned char {
```

Comment:
We need to make sure kUninitialized < kDisable.

I think we can switch to kUninitialized=0, kDisable=1, etc. I don't think we persistent it anywhere.